server/users/userModel.js

```diff
@@ -0,0 +1,32 @@
+var model = require('mongoose').model;
+var Schema = require('mongoose').Schema;
+
+// represents all users of the app, both gifters and giftees 
+var UserSchema = new Schema({
+  birthdate: {
+    // may need to change to Date, using String bc we don't know what FB will give us
+    type: String,
+    required: true
```

Comment:
We should remove this as being required. Some users might not have birthdays from FB.